min_k8s_version/Tiltfile

```diff
@@ -0,0 +1,24 @@
+def min_k8s_version(min_version):
+
+   #This relies on the fact that the shell command returning something in the form 'Server Version: vX.X.X'
+    version_str =  str(local('kubectl version --short| grep "Server Version"', quiet=True))
+   #Get the string, separate it at the colon, remove the 'v' and any spaces
+    version_str = version_str.split(':')[1].replace('v','').strip()
+  
+  #David Rubin's helper function returns the version in the form [X, X, X]
+    cur_ver = _version_tuple(version_str)
+    min_ver = _version_tuple(min_version)
+    
+    if cur_ver < min_ver:
+        print(" ---------------------------------------------------------- ")
+        print("|                                                          |")
+        print("|            Minimum Kubernetes Version Not Met!           |")
+        print("|                Refer link below to update                |")
+        print("| https://kubernetes.io/docs/tasks/tools/install-kubectl/  |")
```

Comment:
this link isn't quite right. i think you're confusing the kubectl version (the local tool you use to access the cluster) with the kubernetes version (the particular server-side components)

in this case, the instructions to upgrade the cluster would depend a lot on what cluster you're using (docker for mac, etc).

It might be better to print out: cur_ver, min_ver, and k8s_context(), to catch the case where the user is connected to a different cluster than they expected